packages/core/src/Renderer.ts

```diff
@@ -370,6 +370,9 @@ export class Renderer extends AbstractRenderer
 
     /**
      * Add a new system to the renderer.
+     *
+     * @extends PIXI.AbstractRenderer#addSystem
```

Comment:
You can remove @extends flag, it's not functional here.

Reply:
Removed.